tests/share/changes/test_change.py

```diff
@@ -193,43 +195,42 @@ def test_can_delete_work(self, john_doe, normalized_data_id):
         graph = ChangeGraph.from_jsonld({
             '@graph': [{
                 '@id': '_:abc',
-                '@type': 'link',
+                '@type': 'identifier',
                 'url': 'https://share.osf.io/faq',
-                'type': 'provider',
             }, {
-                '@id': '_:456',
-                '@type': 'throughlinks',
-                'link': {'@id': '_:abc', '@type': 'link'},
+                '@id': '_:def',
+                '@type': 'workidentifier',
                 'creative_work': {'@id': '_:789', '@type': 'preprint'},
+                'identifier': {'@id': '_:abc', '@type': 'identifier'}
             }, {
                 '@id': '_:789',
                 '@type': 'preprint',
                 'title': 'All About Cats',
+                'identifiers': [{'@id': '_:def', '@type': 'workidentifier'}]
             }]
         })
 
         change_set = ChangeSet.objects.from_graph(graph, normalized_data_id)
 
-        link, preprint, _ = change_set.accept()
+        preprint = [o for o in change_set.accept() if isinstance(o, Preprint)][0]
```

Comment:
This is a subtle but very important nuance. The order that these are returned in is critically import to the disambiguation process.

There are a few other tests that check this but I'd sleep better know that there are many assertions for it.

See `test_topological_sort_many_to_many` for another example.